javascript-practice/src/js/utils/validateForm.js

```diff
@@ -104,10 +71,15 @@ function validateUrl({ key, value }) {
   try {
     new URL(value);
 
-    formError[key] = '';
+    const fileExtension = value.split('.').pop().toLowerCase();
+    if (['jpg', 'jpeg', 'png'].includes(fileExtension)) {
+      formError[key] = '';
+    } else {
+      formError[key] = `${key} must be a valid image URL (jpg, jpeg, or png).`;
+    }
   } catch (error) {
     formError[key] = `${key} must be a valid URL.`;
-  } 
+  }
 }
 
 const validationSchema = {
```

Comment:
I see that you only use this variable for validateForm, why do you don't define it in this function?